js-packages/@prestojs/viewmodel/src/fields/Field.ts

```diff
@@ -7,9 +7,16 @@ import { AsyncChoicesInterface } from './AsyncChoices';
  */
 export interface FieldProps<T> {
     /**
-     * True if field is required when creating or updating a model
+     * Is this field required when saving a record?
      */
-    required?: boolean;
+    blank?: boolean;
```

Comment:
Comment doesn't match the name, should be more explicit

"Is this field allowed to be assigned a blank (null, undefined, '') value?"